commit 1e7aec82f43d
Author: Martin Stransky <stransky@redhat.com>
Date:   Mon Jul 9 13:18:59 2018 +0200

    Bug 1474265 - Add missing semicolon at DBusRemoteClient.cpp. r=jhorak, a=jcristau
    
    MozReview-Commit-ID: HIz4gZOzfOE
    
    --HG--
    extra : source : f1f6e434d3a86d729a39aabf00e8c057c794bed5
    extra : intermediate-source : f27de60fff477d7b61b5509fc2c9c9c098f5b953
---
 widget/xremoteclient/DBusRemoteClient.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git widget/xremoteclient/DBusRemoteClient.cpp widget/xremoteclient/DBusRemoteClient.cpp
index 1919ad027a14..6b242f075913 100644
--- widget/xremoteclient/DBusRemoteClient.cpp
+++ widget/xremoteclient/DBusRemoteClient.cpp
@@ -142,7 +142,7 @@ DBusRemoteClient::GetRemoteDestinationName(const char *aProgram,
       (bool (*)(const char *, DBusError *))
       dlsym(RTLD_DEFAULT, "dbus_validate_bus_name");
     if (!sDBusValidateBusName) {
-      return false
+      return false;
     }
 
     if (!sDBusValidateBusName(aDestinationName.get(), nullptr)) {
